README.md

```diff
@@ -95,26 +87,26 @@ You can use `SearchManager::add()` method to add filter or use specific methods
 like `value()`, `like()` etc. for in built filters.
 
 If you do not want to clutter your `initialize()` method with search config you
-can instead add a `searchConfiguration()` method to the table class. The behavior
-will look if such a method exists and if present use it to get the search manager
-instance from it. This method **must** return a search manager instance.
-
-If you want to change the name of the method, or have multiple methods and
-switch between them, you can configure the name of the method by setting the
-behaviors option `searchConfigMethod` to the name of the method you want.
+can instead add a `searchManager()` method to the table class and reutrn a search
+manager instance.
 
 ```php
-use Search\Manager;
-
 class ExampleTable extends Table {
+    public function initialize(array $config)
+    {
+        parent::initialize($config);
 
-    public function searchConfiguration()
+        // Add the behaviour to your table
+        $this->addBehavior('Search.Search');
+    }
+
+    public function searchManager()
     {
-        $search = new Manager($this);
+        $searchManager = $this->behaviors()->Search->searchManager();
```

Comment:
I believe you can also directly do 

    $this->searchManager()

This is typehinted in the IDE due to the IdeHelper and the loaded behavior :)